pages/htmlresources.md

```diff
@@ -76,15 +76,24 @@ pip install http-parser
 ```
 
 ### Generating the couchapp
-Using `couchapp generate` command you will be able to generate the structure of a couchapp, but first, you need to create a folder with the app name and then, access to it.
+Now you should be in the _couchapp_ folder. Using `couchapp generate` command you will be able to generate the structure of a couchapp, but first, you need to create a folder with the app name and then, access to it.
```

Comment:
Sure. I'll try to be more clear in framing the content too. Thanks for the review.